### DISCUSSION TITLE
svg over page img

### DISCUSSION THREAD

**Discussion by jurgenizer** [ts1/flipbook-vue#106] [General]
Thank you for making this wonderful module!
I would like to overlay an SVG over each of my page` <img>`, using the CSS ::after pseudo-element. It does not seem possible for `<img> `tags in general. Is there another way I could achieve an SVG overlay over each page in flipbook-vue?

**Comment by ts1** [comment]
Thank you for your kind words about the module!

Each page in flipbook-vue not only serves as an `<img>` but also as a `background-image` while flipping. Therefore, simply using `::after` won't work in this case.

Currently, the module only supports image URLs for pages. However, one workaround could be to create a composite image with the SVG overlay and then convert it to a Blob URL to use as a page.

I hope this helps!

**Reply by jurgenizer** [reply]
Thank you very much for replying so soon! I will try making the composite image and Blob URL.
